src/Client/Web/Playground/Bit.Client.Web.BlazorUI.Playground/Web/Pages/Components/Rating/BitRatingDemo.razor.cs

```diff
@@ -33,7 +56,7 @@ public partial class BitRatingDemo
             },
             new ComponentParameter()
             {
-                Name = "DefaultRating",
+                Name = "DefaultValue",
                 Type = "double",
                 DefaultValue = "0",
                 Description = "Default rating. Must be a number between min and max. Only provide this if the Rating is an uncontrolled component; otherwise, use the rating property.",
```

Comment:
Rename `Rating` to `Value` in line 101 and `RatingChanged` to `ValueChanged` in line 108